core/App/ehsm_provider.cpp

```diff
@@ -810,4 +811,46 @@ ehsm_status_t ExportDataKey(ehsm_keyblob_t *cmk,
         return EH_OK;
 }
 
+std::string create_apikey()
+{
```

Comment:
don't return string, follow the above APIs, return ehsm_status_t

Reply:
It will be change